File: Python/4-Triangulo.py
def main():
    med = int(input("Medida: "))
    crear(med)
def crear(med):
    esp = med-1
    print("med: "+str(med)+"\nesp: "+str(esp))
    for i in range(med):
        #for para espacios
        for j in range(esp):
            print(" ",end="")
        print("* "*(med-esp))
        esp=esp-1
main()

File: c++/4-Triangulo.cpp
#include <iostream> //Componente para operaciones de entrada/salida de datos
using namespace std; //Componente para flujo de entrada y salida

    //Metodos
    void Crear(int med);

    int main(){
        int med;
        cout << "Ingrese la medida: ";
        cin >> med;
        Crear(med);
    }
    void Crear(int med){
        int esp;
        esp = med-1;
        cout << "med: " << med << "\nesp: " << esp << "\n";
        for (int i = 0; i < med; i++){
            for (int j = 0; j < esp; j++)
            {
                cout << " ";
            }
            esp--;
            for (int k = 1; k < (med-esp); k++)
            {
                cout << "* ";
            }
            cout << "\n";
        }
        
    }
